data/scripts/combine_projects.py

```diff
@@ -43,89 +43,106 @@ def get_census_tract(lat, long):
 
 
 projects = []
+planned_projects = []
 
 jsonreader_eia = ""
 
-print("Reading ABP data...")
-with open(f"../raw/{IL_ABP_FILE}") as report_3_file:
-    datareader_report_3 = csv.DictReader(report_3_file)
+for project_type in ("energized", "planned"):
+    print("combining", project_type, "projects")
+    print("Reading ABP data...")
 
-    for row in datareader_report_3:
-        if row["Census Tract Code"] == "N/A" or row["Project Size AC kW"] == "":
-            continue
-        
-        cur_project = {}
+    file_suffix = ""
+    if project_type == "planned":
+        file_suffix = "_planned"
 
-        cur_project["source_file"] = IL_ABP_FILE
-        cur_project["kw"] = clean_number(row["Project Size AC kW"])
-        cur_project["census_tract"] = row["Census Tract Code"]
-        cur_project["category"] = row["CEJA Category"]
-        cur_project["energization_date"] = row["Part II Application Verification/ Energization Date"]
+    with open(f"../raw/{IL_ABP_FILE}{file_suffix}.csv") as report_3_file:
+        datareader_report_3 = csv.DictReader(report_3_file)
 
-        if row["CEJA Category"] == "CS":
-            cur_project["category"] = "CS"
-        else:
-            cur_project["category"] = get_category(row["Project Size AC kW"])
-
-        cur_project["county"] = row["Census Tract Code"][2:5]
-
-        projects.append(cur_project)
+        for row in datareader_report_3:
+            if row["Census Tract Code"] == "N/A" or row["Project Size AC kW"] == "":
+                continue
+            
+            cur_project = {}
 
-print("Reading IL SFA data...")
-with open(f"../raw/{IL_SFA_FILE}") as ilsfa_file:
-    datareader_ilsfa = csv.DictReader(ilsfa_file)
+            cur_project["source_file"] = IL_ABP_FILE
+            cur_project["kw"] = clean_number(row["Project Size AC kW"])
+            cur_project["census_tract"] = row["Census Tract Code"]
+            if project_type == "planned":
+                cur_project["energization_date"] = row["Scheduled Energization Date"]
```

Comment:
I try to avoid magic strings, so I'd consider making a constant for the available columns in your data, as otherwise someone could slip in a typo on one specific check

Reply:
can you say more here? this script is combining 3 different CSVs and aligning the appropriate columns into a shared form

Reply:
What I mean is that hard coding `energization_date` and `Scheduled Energ...` is dangerous, since someone could be referencing those elsewhere, make a typo, and break something. Constants, like `OrigCols.EnegDate` prevent that, since the variable would bomb out if you have a typo.

Reply:
ah, i understand. that's easier to do in javascript due to the way they interpret json data. it would take some code to turn each of these string values in the python dict to act as variables, so probably not worth it.